.gitignore

```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+.DS_Store
+package-lock.json
```

Comment:
do we need to ignore npm or node modules? e.g. https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
Done